app/sidekiq/evss/disability_compensation_form/form526_document_upload_failure_email.rb

```diff
@@ -0,0 +1,108 @@
+# frozen_string_literal: true
+
+require 'va_notify/service'
+
+module EVSS
+  module DisabilityCompensationForm
+    class Form526DocumentUploadFailureEmail < Job
+      STATSD_SENT_METRIC_KEY = 'api.form_526.veteran_notifications.document_upload_failure_email.success'
+      STATSD_EXHAUSTED_METRIC_KEY = 'api.form_526.veteran_notifications.document_upload_failure_email.exhausted'
+
+      # retry for one day
+      sidekiq_options retry: 14
+
+      sidekiq_retries_exhausted do |msg, _ex|
+        job_id = msg['jid']
+        error_class = msg['error_class']
+        timestamp = Time.now.utc
+        form526_submission_id, supporting_evidence_attachment_guid = msg['args']
+
+        # Job status records are upserted in the JobTracker module
+        # when the retryable_error_handler is called
+        form_job_status = Form526JobStatus.find_by(job_id:)
+        bgjob_errors = form_job_status.bgjob_errors || {}
+        new_error = {
+          "#{timestamp.to_i}": {
+            caller_method: __method__.to_s,
+            timestamp:,
+            form526_submission_id:,
+            supporting_evidence_attachment_guid:
+          }
+        }
+        form_job_status.update(
+          status: Form526JobStatus::STATUS[:exhausted],
+          bgjob_errors: bgjob_errors.merge(new_error)
+        )
+
+        Rails.logger.warn(
+          'Form526DocumentUploadFailureEmail retries exhausted',
+          { job_id:, timestamp:, form526_submission_id:, error_class:, supporting_evidence_attachment_guid: }
+        )
+
+        StatsD.increment(STATSD_EXHAUSTED_METRIC_KEY)
+      rescue => e
+        ::Rails.logger.error(
+          'Failure in Form526DocumentUploadFailureEmail#sidekiq_retries_exhausted',
+          {
+            messaged_content: e.message,
+            job_id:,
+            submission_id: form526_submission_id,
+            supporting_evidence_attachment_guid:,
+            pre_exhaustion_failure: {
+              error_class:
+            }
+          }
+        )
+        raise e
+      end
+
+      def perform(form526_submission_id, supporting_evidence_attachment_guid)
+        @notify_client ||= VaNotify::Service.new(Settings.vanotify.services.benefits_disability.api_key)
+        submission = Form526Submission.find(form526_submission_id)
```

Comment:
Because nothing is being accessed from `submission` in this method, you could just pass `form526_submission_id` to `send_notification_mailer` and then retrieve the record in that method.